lua/pac3/editor/client/popups_part_tutorials.lua

```diff
@@ -0,0 +1,1180 @@
+			"Size affects the after-images projected around the center, which serve as a base for the effect."
+		},
+
+		["shake"] = {
+			tooltip = "shakes nearby viewers\' camera",
+			popup_tutorial =
+			"This part applies a shake that uses the camera\'s position to take effect. For that reason, it may be nullified by certain third person addons, as well as the pac3 editor camera. You can still temporarily disable it to preview your shakes."
+		},
+
+		["gesture"] = {
+			tooltip = "plays a gesture",
+			popup_tutorial =
+			"Gestures are a type of animation usually added as a layer on top of other animations, this part tries to play one but not all animations listed are gestures, so it might not work for most."
+		},
+
+		["damage_zone"] = {
+			tooltip = "deals damage in a zone",
+			popup_tutorial =
+			"This part tries to deal hitbox damage via the server. It may or may not be allowed because of server settings (pac_sv_damage_zone) and client consents (pac_client_damage_zone_consent), etc. Server owners can add or remove entity classes that can be damaged with pac_damage_zone_blacklist_entity_class, pac_damage_zone_whitelist_entity_class commands.\n"..
+			"Among NPCs it should include VJ and DRG base NPCs, but only if they have npc_ or drg_ in their name\n"..
+			"Most shapes should be self-explanatory but you can use the preview function to see what it should cover. There are some settings for raycasts which could come in handy for some niche use cases even if the basic ones you'll use most of the time (box, sphere, cone from spheres, ray) will not really use these."..
+			"There are certain special damage types. the dissolves can disintegrate entities but can be restricted in the server, prevent_physics_force suppresses the corpse force, removenoragdoll removes the corpse.\n"..
+			""
+		},
+
+		["health_modifier"] = {
+			tooltip = "modifies your health, armor",
+			popup_tutorial =
+			"This part allows you to quickly change your max health, max armor, damage multiplier taken, and has the possibility to give you extra health bars that absorb damage before the main health gets damaged.\n"..
+			"For the extra bars, you need to set a layer priority to pick which ones get damaged first. Outer ones are higher layer values. But they're still invisible for now... events and proxies will come later...\n"..
+			"The part's usage may or may not be allowed by the server."
+		}
+
+	}
+	--print("we have defined the pace.TUTORIALS.PartInfos", pace.TUTORIALS.PartInfos)
+
+	for i,v in pairs(pace.TUTORIALS.PartInfos) do
+		--print(i,v)  
+		if pace.PartTemplates then
+			if pace.PartTemplates[i] then
+				pace.PartTemplates[i].TutorialInfo = v
+			end
+		end
+	end
+end
+
+local motd_cvar = CreateConVar("pac_show_message_on_startup", "2", {FCVAR_ARCHIVE}, "Whether to show the update MOTD when you load in the game")
+
+
+function pac.OpenMOTD(from_initial_startup)
```

Comment:
I don't think this is needed, you're forcing everyone a extra MOTD in their face and then requiring a second popup to confirm if they've read it. This should probably be removed

Reply:
I knew it was gonna be controversial but I simply saw fit that, given how big the additions were, there should be a more direct way to inform server owners and users, because the principle of proportionality. I have a better solution though. I'll just use the gmod's AddLegacy notifications in the corner. You're right. Even if it's just once, it's a bother.